build.gradle.kts

```
@@ -117,46 +118,43 @@ tasks.javadoc {
 /* ******************** publishing ******************** */
 
 publishing {
-    publications.register<MavenPublication>("extensionSdk") {
-        from(components["java"])
+    publications {
+        create<MavenPublication>("extensionSdk") {
+            pom {
+                name.set(project.name)
+                description.set(project.description)
+                url.set(project.metadata.organization!!.url)
+                licenses {
+                    project.metadata.license
+                }
+                developers {
+                    project.metadata.developers
+                }
+                scm {
+                    connection.set("scm:git:git://github.com/hivemq/${project.metadata.github!!.repo}.git")
+                    developerConnection.set("scm:git:ssh://github.com/hivemq/${project.metadata.github!!.repo}.git")
+                    url.set("http://github.com/hivemq/${project.metadata.github!!.repo}/")
+                }
+            }
```

```suggestion
            from(components["java"])
```
all this is done by the metadata plugin to be compatible with maven central out of the box (which is one reason that this gradle plugin exists)
Also, the `from(components["java"])` was deleted